arbor/execution_context.cpp

```diff
@@ -16,7 +16,7 @@ namespace arb {
 
 execution_context::execution_context(const proc_allocation& resources):
     distributed(make_local_context()),
```

Comment:
I guess it doesn't really make sense to set process affinity for the local context?

Reply:
We could, but is this really for performance-conscious users?

Reply:
I'd prefer to have the same behaviour for a program on single process regardless of whether it was compiled with MPI, ymmv.

Reply:
Yeah, if someone made the effort to pass `bind_proc=true`, we should honour their wishes, whatever we might think.

Reply:
_but_, consider what will happen: we will distribute 1 item across the available mask _without_ singlification (that's done only for threads). That's a no-op.

Reply:
true, it doesn't seem that useful